packages/neuron-ui/src/components/MultisigAddressCreateDialog/hooks.ts

```diff
@@ -117,17 +128,18 @@ export const useViewMultisigAddress = ({
   const [multisigAddress, changeMultisigAddress] = useState('')
   useEffect(() => {
     if (step === Step.viewMultiAddress) {
-      createMultisigAddress({
-        r,
-        m,
-        n,
-        addresses,
-        isMainnet,
-      }).then(res => {
-        if (isSuccessResponse(res) && res.result) {
-          changeMultisigAddress(res.result)
-        }
-      })
+      try {
+        const address = getMultisigAddress(
+          addresses.map(v => addressToScript(v).args),
```

Comment:
Should verify if they are all `secp256k1 address`s

Reply:
I only validate required fields but I do not validate fields correctly. I will add validation for correctness.

Reply:
https://github.com/nervosnetwork/neuron/pull/2368/commits/7f4a0c0c062ffa1e56481feb39d327facd7ff043